### PR TITLE
The link to the specific tab for private modules has been updated

### PR DESCRIPTION
**Pull request by jefflembeck**

### CODE REVIEW

File: signup-private-modules.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ require('./lib/sharedNemo').then(function(nemo) {
     bail: true
   }, function(t) {
     return P.all([
-      nemo.driver.get(urlOf('/settings/billing')),
+      nemo.driver.get(urlOf('/settings/billing#payment-information')),
```
Comment (aredridel) [inline signup-private-modules.js:13]: Oh interesting. And you have to hit the hash to open the form?

Comment (jefflembeck) [inline signup-private-modules.js:13]: To open that specific part, yes. Otherwise it takes you to your account overview.

Comment (aredridel) [inline signup-private-modules.js:13]: Okey doke. Weirds me out to see in a test, since I'm used to HTTP tests. Makes sense though.

Comment (aredridel) [inline signup-private-modules.js:13]: Also: web 2.0 is weird.

**Pull request merged**